Makefile

```diff
@@ -12,21 +12,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Use the native vendor/ dependency system
+export GO15VENDOREXPERIMENT=1
```

Comment:
Maybe not yet. With go1.6, it's still possible to explicitely set the var to `0`, so I think we should have it for now. When we require go1.7, we may drop it